certified-connectors/RamQuest Actions - UAT/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1557 @@
+        "foreign": {
+          "description": "Gets or Sets Foreign",
+          "type": "boolean"
+        }
+      }
+    },
+    "PubCS.TitleOrder.Api.Models.Irs1099": {
+      "description": "",
+      "type": "object",
+      "properties": {
+        "id": {
+          "format": "double",
+          "description": "Gets or Sets Id",
+          "type": "number"
+        },
+        "propertyOrServicesReceived": {
+          "description": "Gets or Sets PropertyOrServicesReceived",
+          "type": "string"
+        },
+        "overrideAutoProceedDistribution": {
+          "description": "Gets or Sets OverrideAutoProceedDistribution",
+          "type": "boolean"
+        },
+        "generate1099": {
+          "description": "Gets or Sets Generate1099",
+          "type": "boolean"
+        },
+        "allocationPercent": {
+          "format": "double",
+          "description": "Gets or Sets AllocationPercent",
+          "type": "number"
+        },
+        "allocationAmount": {
+          "format": "double",
+          "description": "Gets or Sets AllocationAmount",
+          "type": "number"
+        }
+      }
+    },
+    "PubCS.TitleOrder.Api.Models.Seller": {
+      "description": "",
+      "type": "object",
+      "properties": {
+        "attributes": {
+          "description": "Gets or Sets Attributes",
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          }
+        },
+        "sequence": {
+          "format": "int32",
+          "description": "Gets or Sets Sequence",
+          "type": "integer"
+        },
+        "businessName": {
+          "$ref": "#/definitions/PubCS.TitleOrder.Api.Models.PartyReference"
+        },
+        "primaryName": {
+          "$ref": "#/definitions/PubCS.TitleOrder.Api.Models.PartyReference"
+        },
+        "secondaryName": {
+          "$ref": "#/definitions/PubCS.TitleOrder.Api.Models.PartyReference"
+        },
+        "relationship": {
+          "description": "Gets or Sets Relationship",
+          "type": "string"
+        },
+        "currentAddress": {
+          "$ref": "#/definitions/PubCS.TitleOrder.Api.Models.TitleOrderAddress"
+        },
+        "forwardingAddress": {
+          "$ref": "#/definitions/PubCS.TitleOrder.Api.Models.TitleOrderAddress"
+        },
+        "communications": {
+          "description": "Gets or Sets Communications",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/PubCS.TitleOrder.Api.Models.Communication"
+          }
+        },
+        "realtor": {
+          "$ref": "#/definitions/PubCS.TitleOrder.Api.Models.PartyReference"
+        },
+        "attorney": {
+          "$ref": "#/definitions/PubCS.TitleOrder.Api.Models.PartyReference"
+        },
+        "tax1099": {
+          "$ref": "#/definitions/PubCS.TitleOrder.Api.Models.Irs1099"
+        }
+      }
+    },
+    "PubCS.TitleOrder.Api.Models.Buyer": {
+      "description": "",
```

Comment:
please use a proper description for the response. you can use something like "OK", "Sucess", "Bad request", etc.